### PR TITLE
config,distsql: add config for coprocessor streaming feature

### PR DESCRIPTION
**Pull request by tiancaiamao**
This config is necessary before it's enabled by default.

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -200,15 +201,16 @@ type TiKVClient struct {
...
+	Host:            "0.0.0.0",
+	Port:            4000,
+	Store:           "mocktikv",
+	Path:            "/tmp/tidb",
+	RunDDL:          true,
+	SplitTable:      true,
+	Lease:           "10s",
+	TokenLimit:      1000,
+	EnableChunk:     true,
+	EnableStreaming: false,
```
Comment (zz-jason) [inline config/config.go]: Can we make it as a session variable? Thus we can compare the results between streaming and normal APIs  without restarting tidb-server.

**Comment by jackysp** [comment]
LGTM

**Comment by jackysp** [comment]
LGTM

**Pull request merged**